internal/pkg/tui/installer/state.go

```diff
@@ -268,27 +268,27 @@ func configureAdapter(installer *Installer, opts *machineapi.GenerateConfigurati
 				items := []*components.Item{
 					components.NewItem(
 						"Use DHCP",
-						v1alpha1.DeviceDoc.Describe("dhcp", true),
+						v1alpha1.Device{}.Doc().Describe("dhcp", true),
 						&adapterSettings.Dhcp,
 					),
 					components.NewItem(
 						"Ignore",
-						v1alpha1.DeviceDoc.Describe("ignore", true),
+						v1alpha1.Device{}.Doc().Describe("ignore", true),
 						&adapterSettings.Ignore,
 					),
 					components.NewItem(
 						"CIDR",
-						v1alpha1.DeviceDoc.Describe("cidr", true),
+						v1alpha1.Device{}.Doc().Describe("cidr", true),
 						&adapterSettings.Cidr,
 					),
 					components.NewItem(
 						"MTU",
-						v1alpha1.DeviceDoc.Describe("mtu", true),
+						v1alpha1.Device{}.Doc().Describe("mtu", true),
 						&adapterSettings.Mtu,
 					),
 					components.NewItem(
 						"Route Metric",
-						v1alpha1.DeviceDoc.Describe("dhcpOptions", true),
+						v1alpha1.Device{}.Doc().Describe("dhcpOptions", true),
```

Comment:
Not sure if it helps, but maybe something like:

```go
v1alpha1.Describe[v1alpha1.Device]("dhcpOptions", true)

// or if we define it in the same module
describe[v1alpha1.Device]("dhcpOptions", true)
```
will be more readable here

Reply:
yep, good idea, I will update

Reply:
updated